configs/baremetal.h

```diff
@@ -137,6 +137,8 @@
 
 #define MBEDTLS_OID_C
 #define MBEDTLS_PLATFORM_C
+#define MBEDTLS_CRC_C
+#define MBEDTLS_VALIDATE_AES_KEYS
```

Comment:
@AndrzejKurek , you forgot to rename MBEDTLS_VALIDATE_AES_KEYS to  MBEDTLS_VALIDATE_AES_KEYS_INTEGRITY in this file

Reply:
Done.